packages/core/src/Button/Button.tsx

```diff
@@ -224,7 +224,7 @@ export interface IButtonProps
     React.RefAttributes<unknown> {
   color?: string
   variation?: 'fill' | 'link' | 'outline' | 'plain' | 'subtle' | 'white' | 'lightFill' | 'input'
-  size?: Sizes | Sizes[]
+  size?: Sizes
```

Comment:
are we dropping support for `Sizes[]` this is for when we want it to be responsive e.g., `size={['small', null, 'medium']}`

Reply:
Ah great catch, this is a change I made when messing with the controls for the default story, and missed adding it back/undoing it! Thanks for finding this, I just made a commit to revert it 😄